src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 13px;
+    height: 13px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    margin-right: 8px;
```

Comment:
why 8px? should be 4

Reply:
In the last commit, you deleted the resetting of the margin-right - revert it (it should be 0 for last start, as mentor said in the previous review)